website/docs/components/alert/partials/guidelines/guidelines.md

```diff
@@ -34,41 +34,56 @@ Alert types have an implied hierarchy and should be ordered starting from `page`
 
 ## Color
 
+### Information alerts 
 <Doc::Layout @spacing="12px">
   <Hds::Alert @type="inline" @color="neutral" as |A|>
-    <A.Title>Neutral alert title</A.Title>
-    <A.Description>Lorem ipsum dolar sit amet.</A.Description>
+    <A.Title>Note</A.Title>
+    <A.Description>GitHub Actions doesn’t support names starting with the “GitHub_” prefix. If your secret names include “GitHub_”, you’ll need to rename them for proper syncing secrets.</A.Description>
   </Hds::Alert>
   <Hds::Alert @type="inline" @color="highlight" as |A|>
-    <A.Title>Highlight alert title</A.Title>
-    <A.Description>Lorem ipsum dolar sit amet.</A.Description>
-  </Hds::Alert>
+    <A.Title>Upgrade</A.Title>
+    <A.Description>Looks like you’re getting close to using up free tier. Unlock more integrations and other premium features. [Upgrade now](https://helios.hashicorp.design/).</A.Description>
+      </Hds::Alert>
+</Doc::Layout>
+
+Informational alerts are non-urgent, non-disruptive alerts that don’t require immediate attention.
```

Comment:
I feel like we should mention in the guidance, what this means for assistive technology. So something along the lines of:

```suggestion
Informational alerts are non-urgent, non-disruptive alerts that don’t require immediate attention. This also means from an assistive technology perspective, by default, they will not be announced to the user.
```
From a UX perspective, designers should know the difference between this and the other kinds of alerts and how it impacts AT. @heatherlarsen what do you think?

Reply:
Already addressed in a [previous comment](https://github.com/hashicorp/design-system/pull/2509/files#r1806806357) 🙂

Reply:
lol great minds ✨

Reply:
I am going to resolve this comment!